addons/mass_mailing/static/src/scss/mass_mailing.ui.scss

```diff
@@ -301,6 +301,28 @@ body.editor_enable.o_basic_theme.o_in_iframe {
             display: block;
         }
     }
+    #oe_snippets.o_loaded {
+        transform: translateX(0);
+
+        > .o_we_customize_panel .oe-toolbar {
+            #style.dropdown, #font-size.dropdown {
+                > .dropdown-menu {
+                    overflow-y: scroll;
+                    max-height: 180px;
+                }
+            }
+            // Popper.js manipulates transform and position of the dropdown. In
+            // mass_mailing, its calculations are wrong. These !important rules
+            // removes the wrong values automatically generated by Popper.
+            #justify , #style.dropdown, #font-size.dropdown {
+                > .dropdown-menu {
+                    transform: unset !important;
+                    bottom: 100% !important;
+                    top: unset !important;
```

Comment:
I'm confused, I thought @morm-odoo had done pretty much the exact same fix a couple of months ago. Was it in 16.0 ?

Reply:
@Zinston Yes this fix is done in 16.0 and not in 15.0.

Reply:
Thanks, that clarifies it. Sorry for budding in, carry on :-D